charts/thoras/templates/api-server-v2/deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
       - image: {{ .Values.imageCredentials.registry }}/services:{{ default .Values.thorasVersion .Values.thorasApiServerV2.imageTag }}
         imagePullPolicy: "{{ .Values.imagePullPolicy }}"
         name: thoras-v2-api-server
-        command: ["/app/api-server", "-p", {{ .Values.thorasApiServerV2.containerPort | quote }}]
+        command: ["/app/api-server", "-p", {{ .Values.thorasApiServerV2.containerPort | quote }}, "--certificate-file", "/app/cert.pem", "--key-file", "/app/key.pem"]
```

Comment:
[dust] We've added a bit to the line, a vertical list may make increase readability. Your call!

Reply:
Done!